oatomobile/baselines/torch/dim/train.py

```diff
@@ -180,11 +187,13 @@ def train_step(
   ) -> torch.Tensor:
     """Performs a single gradient-descent optimisation step."""
     # Resets optimizer's gradients.
+    # pdb.set_trace()
     optimizer.zero_grad()
 
+    target_mean = batch["player_future"][..., :2] 
     # Perturb target.
     y = torch.normal(  # pylint: disable=no-member
-        mean=batch["player_future"][..., :2],
+        mean=target_mean,
         std=torch.ones_like(batch["player_future"][..., :2]) * noise_level,  # pylint: disable=no-member
```

Comment:
use `target_mean` instead of `batch["player_future"][..., :2]`

Reply:
@filangel it looks like I'm already using `target_mean`. Can you clarify?